scss/_utilities/_grid.scss

```diff
@@ -6,10 +6,10 @@
 .j-row {
```

Comment:
As far as I remember, the only thing within jetpack using this grid is the settings areas, so we'll be able to eliminate this. grid system.